spec/models/service_template_transformation_plan_task_spec.rb

```diff
@@ -1,5 +1,15 @@
 RSpec.describe ServiceTemplateTransformationPlanTask, :v2v do
   let(:infra_conversion_job) { FactoryBot.create(:infra_conversion_job) }
+  let(:src_ems) { FactoryBot.create(:ems_vmware) }
+  let(:src_cluster) { FactoryBot.create(:ems_cluster, :ext_management_system => src_ems) }
+
+  let(:redhat_ems) { FactoryBot.create(:ems_redhat, :zone => FactoryBot.create(:zone), :api_version => '4.2.4') }
+  let(:redhat_cluster) { FactoryBot.create(:ems_cluster, :ext_management_system => redhat_ems) }
+  let(:redhat_hosts) { FactoryBot.create_list(:host, 1, :ems_cluster => redhat_cluster) }
```

Comment:
Wow I've never heard of create_list before, that is really nice.  Going to have to find everywhere we do `N.times.map { FactoryBot.create() }`...